web/src/components/home/home.js

```diff
@@ -1,6 +1,6 @@
 import SongOfTheWeek from "./subcomponents/song_of_the_week/song_of_the_week";
 import UpcomingEvents from "./subcomponents/upcoming_events/upcoming_events";
-import Welcome from "./subcomponents/welcome/welcome";
+import { Welcome } from "./subcomponents/welcome/welcome";
```

Comment:
Is there a reason to do this? If now, can we revert?

Reply:
yeah, I tried to clean up the export but it does make this import different from the rest. I'll revert